src/models/stores/workspace.ts

```diff
@@ -65,11 +66,13 @@ export const WorkspaceModel = types
         }
       },
 
-      toggleComparisonVisible({override, muteLog = false}: {override?: boolean; muteLog?: boolean} = {}) {
+      toggleComparisonVisible({override, muteLog = false, hidePrimary = false}:
+          {override?: boolean; muteLog?: boolean, hidePrimary?: boolean} = {}) {
         const visible = typeof override !== "undefined" ? override : !self.comparisonVisible;
         self.comparisonVisible = visible;
+        self.hidePrimaryForCompare = hidePrimary;
         if (!visible) {
-          self.comparisonDocumentKey = undefined;
+           self.comparisonDocumentKey = undefined;
```

Comment:
```suggestion
          self.comparisonDocumentKey = undefined;
```
nit: extraneous space

Reply:
Done.